.macos

```diff
@@ -45,35 +48,35 @@ defaults write com.apple.systemuiserver menuExtras -array \
 	"/System/Library/CoreServices/Menu Extras/Clock.menu"
 
 # Set highlight color to green
-defaults write NSGlobalDomain AppleHighlightColor -string "0.764700 0.976500 0.568600"
+defaults write -g AppleHighlightColor -string "0.764700 0.976500 0.568600"
```

Comment:
Sorry for responding so late. Thanks for clarifying! I've rolled back the change on my main branch, as being more explicit is always better :)